pkg/events/pipeline/event.go

```diff
@@ -0,0 +1,185 @@
+package pipeline
+
+import (
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+type Event struct {
+	// 8-byte aligned fields (int, uint, uint64)
+	Timestamp             int    `json:"timestamp"`
```

Comment:
We have getters now... do you plan to isolate the fields (making them private)? If so, Let's introduce the setters as well to fields that require it only.

Reply:
Not at the moment, see above.